### DISCUSSION TITLE
How to trigger datePicker from any element other than its own input-like component?

### DISCUSSION THREAD

**Discussion by MexicanSparker** [ant-design/ant-design#30767] [Q&A]
Hi. I wonder if there is a way to have a button as a trigger for the datePicker component. I don't know if this is possible judging for the numerous examples in the official website but there is no single one that can be triggered from an icon button for example. Is there a way?
Thanks in advance.

**Comment by yoyo837** [comment]
![image](https://user-images.githubusercontent.com/6134547/120128412-7a1f8880-c1f4-11eb-9f6c-2d607496afd6.png)

**Reply by MexicanSparker** [reply]
Excellent. It worked! I would never be able to find out this by myself, thanks a million times.
